web-devtools/global.d.ts

```diff
@@ -0,0 +1,13 @@
+import { DefaultTheme } from "styled-components";
+
+import { darkTheme } from "@kleros/ui-components-library";
+
+declare module "styled-components" {
+  type Theme = typeof darkTheme;
+  export interface DefaultTheme extends Theme {
+    klerosUIComponentsSkeletonBackground: string;
+    klerosUIComponentsSkeletonHighlight: string;
+    klerosUIComponentsVioletPurple: string;
+    klerosUIComponentsLavenderPurple: string;
+  }
+}
```

Comment:
Does it work with this syntax? 👀 If it works this way, I would go with it, so we don't need to update the type every time we add a new key to the theme 🤔
```suggestion
import { DefaultTheme } from "styled-components";

import { DefaultTheme as DevtoolsTheme } from "styles/Theme.tsx";

declare module "styled-components" {
  type Theme = typeof DevtoolsTheme;
  //eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface DefaultTheme extends Theme {}
}
```